kube/resources/resource-defaults.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: LimitRange
```

Comment:
See comment on other PR.

Reply:
this was not supposed to be in this PR 😁